client/src/modules/Score/components/ScoreTable/index.tsx

```diff
@@ -109,6 +113,28 @@ export function ScoreTable(props: Props) {
 
   const [state, setState] = useState(['']);
 
+  useEffect(() => loadInitialData() as any, [activeOnly]);
+
+  useEffect(() => {
+    if (width < 400) {
+      setFixedColumn(false);
+      return;
+    }
+
+    setFixedColumn('left');
+  }, [width]);
+
+  useEffect(() => {
+    setColumns(prevColumns => {
+      const githubColumn = prevColumns.find(el => el.title === 'Github');
```

Comment:
Is there a field named `key` to use it? Because title is not suits for this purpose